src/Termonad/Gtk.hs

```diff
@@ -58,3 +79,17 @@ widgetEq a b = do
     withManagedPtr managedPtrA $ \ptrA ->
       withManagedPtr managedPtrB $ \ptrB ->
         pure (ptrA == ptrB)
+
+-- | Wrapper around 'terminalSetEnableSixel'.  The 'terminalSetEnableSixel' function
+-- is only available starting with vte-0.63. This function has no effect when
+-- compiling against previous versions of vte.
+terminalSetEnableSixelIfExists
+  :: (HasCallStack, MonadIO m, IsTerminal t)
+  => t -- ^ a Terminal
+  -> Bool -- ^ whether to enable SIXEL images
+  -> m ()
+terminalSetEnableSixelIfExists t b = do
+#ifdef VTE_VERSION_GEQ_0_63
+  terminalSetEnableSixel t b
+#endif
```

Comment:
This `terminalSetEnableSixel` function is only available in vte >= 0.63, and ubuntu-20.04 doesn't have a new enough version of VTE.  I used CPP here in order to only call the function if the user is compiling with a new enough VTE.